src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntitiesService.java

```diff
@@ -16,12 +16,12 @@
 import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
```

Comment:
A bit confused - there seems to be no real changes in here. Are they coming in a later PR? Was expecting the trust DTO data to be mapped and saved like in the other PR I reviewed.